public/css/style.css

```diff
@@ -126,16 +131,30 @@ a{
     font-size: 15px;
     background-color: white;
     color:black;
+    bottom: 0;
+    position: fixed;
 }
 .footer h4{
     font-family: 'Poiret One', cursive;
     font-size: 25px;
-    color: rgb(57, 108, 146); 
+    color: rgb(57, 108, 146);
 }
 .footer a:hover{
     color: rgb(57, 108, 146);
     font-size:1.2em;
 }
+.col-sm-6 {
```

Comment:
@Tobi2K  Please do not change predefined classes in bootstrap instead make another class and use it for the purpose